Kernel/Thread.cpp

```diff
@@ -462,6 +446,13 @@ void Thread::set_default_signal_dispositions()
     m_signal_action_data[SIGWINCH].handler_or_sigaction = VirtualAddress((u32)SIG_IGN);
 }
 
+void Thread::push_value_on_user_stack(RegisterDump* registers, u32 value)
```

Comment:
This function doesn't handle `registers` being null.
Therefore the signature should be:
```suggestion
void Thread::push_value_on_user_stack(RegisterDump& registers, u32 value)
```